src/parser.js

```diff
@@ -578,7 +580,9 @@ function mapTryCatchBlock(node, meta) {
   let catchBlock = null;
   if (node.recovery) {
     const recovery = mapBlockStatement(node.recovery, meta);
-    const errorVar = mapLiteral({base: node.errorVariable}, meta);
+    const errorVar = node.errorVariable ?
+      mapLiteral({base: node.errorVariable}, meta) :
+      b.identifier(DEFAULT_CATCH_VARIABLE_NAME);
```

Comment:
I seem to remember decaf having a utility for creating a unique identifier in a particular scope, with an "ideal" name as the base. Am I imagining that?